Admin/FieldDescription.php

```diff
@@ -98,6 +98,10 @@ public function getValue($object)
             $object = $this->getFieldValue($object, $parentAssociationMapping['fieldName']);
         }
 
+        if (isset($this->getFieldMapping()['declaredField'])) {
```

Comment:
Could you please add a comment about why this condition is here?

Something simple like this:

```php
// Support embedded object for mapping
// Ref: http://doctrine-orm.readthedocs.io/projects/doctrine-orm/en/latest/tutorials/embeddables.html
```